assets/scss/app.scss

```diff
@@ -1,8 +1,7 @@
 $path: "../img/";
 
-@import "hof-frontend-toolkit";
+@import "hof-theme-govuk";
 
-@import "./alert.scss";
 @import "./icons.scss";
 @import "./validation.scss";
 @import "./print.scss";
```

Comment:
I had a look and there's no duplication of code between here and `hof-frontend-toolkit`, so yes, these stylesheets are still required.